.gitignore

```diff
@@ -90,3 +90,4 @@ llvm-external-projects/iree-dialects/.cache
 
 # pkgci artifacts
 artifacts/
+third_party/llvm-diff.txt
```

Comment:
please revert the change in `.gitignore`

Reply:
Fixed. Sorry for this. I thought I removed it.